### ISSUE TITLE
Typified by highly variable age of onset

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9884]
**Preferred term label:**
Typified by highly variable age of onset

**Definition (free text, please give PubMed ID)**
Description of conditions in which age of onset is highly variable even in family members who share the same disease-associated variant or variants.

**Parent term (use hpo.jax.org/app)**
[Inheritance modifier HP:0034335](https://hpo.jax.org/app/browse/term/HP:0034335)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034857

**Comment by pnrobinson** [comment]
finalized
